domain/induction/src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/induction/FutureWorkInterests.kt

```diff
@@ -28,7 +28,7 @@ data class WorkInterest(
   val workType: WorkInterestType,
   val workTypeOther: String?,
   val role: String?,
-) : DomainKeyAware {
+) : KeyAwareDomain {
```

Comment:
I'm not sure if this name is better or worse, but it makes more sense within `KeyAwareEntityMapper`, where we now have `KeyAwareChildEntity`.